Cargo.lock

```diff
@@ -11122,7 +11122,7 @@ version = "1.6.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "1f559b464de2e2bdabcac6a210d12e9b5a5973c251e102c44c585c71d51bd78e"
 dependencies = [
- "cfg-if 1.0.0",
+ "cfg-if 0.1.10",
```

Comment:
https://github.com/rust-lang/cargo/issues/5702 in case you're wondering why

it is also possible to revert manually, but I'm not sure if it's worth it

Reply:
woah, good catch! Let me try to revert that

Reply:
hm, on the second thought we already have 0.1.10 so not sure if it's worth it as well